apps/list.c

```diff
@@ -1209,9 +1209,11 @@ static int provider_cmp(const OSSL_PROVIDER * const *a,
 static int collect_providers(OSSL_PROVIDER *provider, void *stack)
 {
     STACK_OF(OSSL_PROVIDER) *provider_stack = stack;
-
-    sk_OSSL_PROVIDER_push(provider_stack, provider);
-    return 1;
+    /*
+	 * If OK - result is the index of inserted data
```

Comment:
Please, no tabs in files they are evil as the misindentation I'm seeing indicates :)

Reply:
Hello Pauli.
Tabs are removed.